API-Samples/C#/ArmBased/Program.cs

```diff
@@ -280,6 +283,21 @@ static string CreateQueryString(IDictionary<string, string> parameters)
             return queryParameters.ToString();
         }
 
+        static string AddExcludedAIs(string excludedAI)
+        {
+            if (String.IsNullOrEmpty(excludedAI))
+            {
+                return "";
+            }
+            var list = excludedAI.Split(',');
+            var result = "";
+            foreach (var item in list)
+            {
+                result += "&excludedAI=" + item;
```

Comment:
Tomer, I think you will get the excludeAI param multiple time while it should be a single key with comma separated value

Reply:
yes , this is the design of this API as it needs to be an ARRAY passed inside query param.
look here : https://github.com/Azure-Samples/media-services-video-indexer/blob/master/API-Samples/C%23/ArmBased/Program.cs

![image](https://github.com/Azure-Samples/media-services-video-indexer/assets/512709/93419fd6-7389-4d89-bb9d-5d2cb55328f1)

Reply:
this code change does not belong to this PR and was contributed by another team member.
this is how this API works , it expects an array of items 
and this is how you pass an array of items in C# when sending them as Query Params.
see : https://stackoverflow.com/a/11100414